packages/amplify-e2e-core/src/utils/nexpect.ts

```diff
@@ -753,7 +753,7 @@ export function nspawn(command: string | string[], params: string[] = [], option
   // For push operations in E2E we have to explicitly disable the Amplify Console App creation
   // as for the tests that need it, it is already enabled for init, setting the env var here
   // disables the post push check we have in the CLI.
-  if (params.length > 0 && params.find((param: string) => param.toLowerCase() === 'push')) {
+  if (params.length > 0 && params.some((param: string) => param.toLowerCase() === 'push')) {
```

Comment:
This change is not related to this PR...just came across it when reading the code